Source/WebCore/style/ElementRuleCollector.cpp

```diff
@@ -584,14 +584,59 @@ bool ElementRuleCollector::containerQueriesMatch(const RuleData& ruleData, const
 
 bool ElementRuleCollector::scopeRulesMatch(const RuleData& ruleData, const MatchRequest& matchRequest)
 {
-    auto queries = matchRequest.ruleSet.scopeRulesFor(ruleData);
+    auto rules = matchRequest.ruleSet.scopeRulesFor(ruleData);
```

Comment:
Lets call this `scopeRules`, as plain `rule` in ElementRuleCollector refers to style rules.